modules/velero-oadp-version-relationship.adoc

```diff
@@ -18,4 +18,6 @@
 | 1.2.2 | link:https://{velero-domain}/docs/v1.11/[1.11] | 4.11 and later
 | 1.2.3 | link:https://{velero-domain}/docs/v1.11/[1.11] | 4.11 and later
```

Comment:
remove "and later" replace with "OCP 4.10 - 4.15"

Reply:
thank you, I've updated that.